api/src/util/entities/AssetCacheItem.ts

```diff
@@ -0,0 +1,16 @@
+import { Headers } from "node-fetch";
+
+export class AssetCacheItem {
+	public Key: string;
+	public FilePath: string;
+	public Headers: any;
+
+	constructor(key: string){
```

Comment:
To prevent redudancy, i would suggest making the key param public in the constructur:
```ts
constructor(public key: string){}
```
instead of
```ts
constructor(key: string){
  this.key = key
}
```

Reply:
i wasn't aware you could do that, thanks